v3/plugins/redhat/rhamt/0.0.19/meta.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+id: redhat.rhamt-vscode-extension
+version: 0.0.19
+type: VS Code extension
+name: rhamt
+displayName: Red Hat Application Migration Toolkit
+title: Red Hat Application Migration Toolkit
+description: Application Migration and Modernization using RHAMT
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: redhat
+repository: https://github.com/windup/rhamt-vscode-extension
+category: Other
+firstPublicationDate: "2020-01-27"
+spec:
+  endpoints:
+  - name: "configuration-endpoint"
+    public: true
+    targetPort: 61436
+    attributes:
+      protocol: http
+  - name: "report-endpoint"
+    public: true
+    targetPort: 61435
+    attributes:
+      protocol: http
+  containers:
+  - image: "docker.io/windup3/rhamt-vscode-extension:java8"
```

Comment:
@johnsteele how is this image built? if we want to include RHAMT ootb in CRW 2 we need to know how this sidecar is created and make a RH-compatible one.